app/models/controlled_vocabulary.rb

```diff
@@ -30,7 +30,7 @@ def initialize(key: nil)
 
   # Method returning all terms within the vocabulary
   # This should be overridden
-  def all
+  def all(_scope = nil)
```

Comment:
what is this for?

Reply:
It's...unrelated. It popped up because of a bug in a previous implementation. It's definitely broken and doesn't fulfill the expected interface, but we have nothing testing it. I'll change it back and make an issue.